src/styles.css

```diff
@@ -79,3 +79,198 @@ body {
         background-position: 600px bottom;
     }
 }
+
+/* Toastr CSS */
```

Comment:
Instead of including the CSS, could we just include the one that comes from the package? That way, if the CSS gets updated in a new version of `ngx-toastr`, we'll get to inherit them.

```diff
diff --git a/angular.json b/angular.json
index e0df609..c491ef7 100644
--- a/angular.json
+++ b/angular.json
@@ -21,6 +21,7 @@
               "src/favicon.ico"
             ],
             "styles": [
+              "node_modules/ngx-toastr/toastr.css",
               "node_modules/bootstrap/dist/css/bootstrap.min.css",
               "node_modules/patternfly/dist/css/patternfly.min.css",
               "node_modules/patternfly/dist/css/patternfly-additions.min.css",
```